src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -108,53 +279,62 @@ private void launchNode(SchedulerDriver driver, Offer offer,
           .build();
```

Comment:
Not yours, but why are we passing the command in using TaskInfo.data instead of CommandInfo.value?